src/RosClientNode.hpp

```diff
@@ -130,7 +130,7 @@ class RosClientNode : public QObject {
    */
   template <typename T>
   void register_local_msg_type(
-      const std::string& from_topic, const std::string& to_topic, uint8_t max_publish_rate_hz=10) {
+      const std::string& from_topic, const std::string& to_topic, float max_publish_rate_hz=10) {
```

Comment:
this is a great change imo--what if we just make this a `double`, since it turns into one eventually anyway?

Reply:
Sure, I'll change to a double and also pull the latest changes in